CHANGELOG.md

```diff
@@ -1,5 +1,147 @@
+- fix: Update attributes `copy_protection_enabled`, `pit_enabled` and `encryption_at_rest_enabled` in the  resource `mongodbatlas_backup_compliance_policy` to be Optional [\#1803](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1803) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- fix: Incompatible schema defined for `mongodbatlas_backup_compliance_policy` [\#1799](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1799) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- fix: Fix `mongodbatlas_clusters` plural data source to set `auto_scaling_disk_gb_enabled` attribute correctly [\#1722](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1722) ([maastha](https://github.com/maastha))
+
+**Closed Issues**
+
+- CANNOT_DISABLE_PIT_WITH_BACKUP_COMPLIANCE_POLICY  [\#1855](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1855)
+- Error creating user `mongodbatlas_database_user` [\#1852](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1852)
+- Serverless `aws_vpc_endpoint` creation fails [\#1826](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1826)
+- [Bug]: Changing Scope block in the databaseuser resource results in a replacement [\#1821](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1821)
+- [Bug]: `oplog_min_retention_hours` is not expected here [\#1818](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1818)
+- Unable to add encryption at rest through terraform [\#1766](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1766)
+- Provider produced inconsistent result after apply [\#1708](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1708)
+- Error: json: cannot unmarshal number 9501614080 into Go struct field CloudProviderSnapshot.storageSizeBytes of type int  [\#1341](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1341)
+- [Query] Unable to find the `endpoint_service_id` [\#1281](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1281)
+
+**Internal Improvements**
+
+- chore: Changes from env variable to input in Import GPG key [\#1898](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1898) ([oarbusi](https://github.com/oarbusi))
+- chore: Upgrades `mongodbatlas_custom_db_role` resource to auto-generated SDK [\#1896](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1896) ([lantoli](https://github.com/lantoli))
+- chore: Bump github.com/hashicorp/terraform-plugin-sdk/v2 from 2.31.0 to 2.32.0 [\#1893](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1893) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Bump github.com/aws/aws-sdk-go from 1.50.3 to 1.50.7 [\#1892](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1892) ([dependabot[bot]](https://github.com/apps/dependabot))
```

Comment:
recommend bumps together at the end of the section

Reply:
Thanks for the suggestion. Done